lib/theme/index.js

```diff
@@ -51,6 +51,28 @@ export const generateTheme = ({
       },
     },
 
+    graystyle: {
```

Comment:
Let's not define a new button style, we don't have a matching counterpart in the design system (figma).